### DISCUSSION TITLE
nocodb后端接口怎么制作docker镜像，修改了后端接口怎么制作自己的docker镜像

### DISCUSSION THREAD

**Discussion by linqianppb** [nocodb/nocodb#8450] [General]
### Please confirm if feature request does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

无

### Suggested Solution

无

### Additional Context

![image](https://github.com/nocodb/nocodb/assets/5062416/49b2dcac-dd9f-4314-b221-f2856b3d4ea0)

**Comment by fulldecent** [comment]
你用`docker`的时候要多加一个`-p 8080:8080`。

如：

```sh
docker run -p 8080:8080 my-app-image
```
